snippets/widgets/character_map.py

```diff
@@ -1,6 +1,6 @@
 from gi.repository import Gtk, Gdk
 import cairo
-import pango
+import Pango
```

Comment:
I'm working on removing the flake8 errors right now, only 3 files are left

On Sun, Jul 24, 2016 at 7:03 PM Dave Crossland <notifications@github.com>
wrote:

> In snippets/widgets/character_map.py
> <https://github.com/sugarlabs/edit-fonts-activity/pull/74#discussion_r71988674>
> :
>
> > @@ -1,6 +1,6 @@
> >  from gi.repository import Gtk, Gdk
> >  import cairo
> > -import pango
> > +import Pango
>
> This is the old import style
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sugarlabs/edit-fonts-activity/pull/74/files/664c54eb3356ae3042184245f15a3a1ca7d52fc0..755f9836d1b69ca9af801fcdd0c95abc9b15e22c#r71988674>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACiNKvjP2N3v4UFSNvMGyBGTW85IkIEAks5qY2mMgaJpZM4JL066>
> .
>